### PR TITLE
20231109 1122 credential class

### PR DESCRIPTION
**Pull request by Firstyear**
Relates #1122 - adds the needed policy for credential types. 

Checklist

- [ x ] This pr contains no AI generated code
- [ x ] cargo fmt has been run
- [ ] cargo clippy has been run
- [ x ] cargo test has been run and passes
- [ ] book chapter included (if relevant)
- [ ] design document included (if relevant)

### CODE REVIEW

**Review by yaleman — CHANGES_REQUESTED** [review]

File: libs/client/src/group.rs [hunk 1]
```diff
@@ -21,6 +21,18 @@ impl KanidmClient {
         .await
     }
 
+    pub async fn group_account_policy_credential_type_minimum(
```
[the remark this answers is not in the document]
Comment (Firstyear) [reply libs/client/src/group.rs:24]: Yep, that's a reasonable change :)

File: server/web_ui/build_wasm.sh [hunk 2]
```diff
@@ -32,7 +32,7 @@ if [ -f ./pkg/.gitignore ]; then
 fi
 
 # copy the shared static things
-rsync -av shared/static/* shared/static/* pkg/
+rsync -av shared/static/* admin/static/* user/static/* login_flows/static/* pkg/
```
Comment (yaleman) [inline server/web_ui/build_wasm.sh:35]: we still need the shared things here, it includes the "shared" javascript, icons and stuff

**Review by yaleman — CHANGES_REQUESTED** [review]
LGTM but we still need the shared things 😄

File: server/web_ui/build_wasm.sh (hunk 2 above)
Comment (yaleman) [reply server/web_ui/build_wasm.sh:35]: ignore me I R derp

**Review by yaleman — APPROVED** [review]
I R Derp. LGTM.

**Pull request merged**